docs/setup/storage/storage.md

```diff
@@ -361,3 +362,44 @@ export ZEPPELIN_NOTEBOOK_MONGO_AUTOIMPORT=true
 
 #### Import your local notes automatically
 By setting `ZEPPELIN_NOTEBOOK_MONGO_AUTOIMPORT` as `true` (default `false`), you can import your local notes automatically when Zeppelin daemon starts up. This feature is for easy migration from local file system storage to MongoDB storage. A note with ID already existing in the collection will not be imported.
+
+## Notebook Storage in GitHub
+
+To enable GitHub tracking, uncomment the following properties in `zeppelin-site.xml`
+
+```sh
+<property>
+  <name>zeppelin.notebook.git.remote.url</name>
+  <value>git@github.com:mohamagdy/zeppelin-notebooks.git</value>
+  <description>remote Git repository URL</description>
+</property>
+
+<property>
+  <name>zeppelin.notebook.git.remote.username</name>
+  <value>token</value>
+  <description>remote Git repository username</description>
+</property>
+
+<property>
+  <name>zeppelin.notebook.git.remote.access-token</name>
+  <value>1b29e853bc73b90af0d280233101f91b9105d7cb</value>
+  <description>remote Git repository password</description>
+</property>
+
+<property>
+  <name>zeppelin.notebook.git.remote.remote</name>
```

Comment:
This supposed to be `zeppelin.notebook.git.remote.origin`?

Reply:
Good catch 👍 ... Fixed in https://github.com/apache/zeppelin/pull/2700/commits/2c1cf743b54861e9bd53e483f3c873a77dbba1b1